server/userAuth.js

```diff
@@ -81,6 +104,18 @@ router.get('/auth/redirect', passport.authenticate(authStrategy, {failureRedirec
   res.redirect(req.session.authRedirect || formatUrl('/'))
 })
 
+router.get('/metadata', function (req, res) {
+  if (isSamlAuth) {
+    res.type('application/xml')
+    res.send((samlStrategy.generateServiceProviderMetadata(
+      process.env.SAML_MD_DECRYPTION_CERT,
+      process.env.SAML_MD_SIGNING_CERT
```

Comment:
I would change this to SAML_SP_SIGNING_CERT

Reply:
Thank you!